.changeset/rich-bees-raise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@base-org/build-onchain-apps': minor
+---
+
+Add new build-onchain-apps CLI for onchain app creation
```

Comment:
Remember to prefix with either `fix:`, `feat:`, `chore:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:` any of the mention in the changelog.